src/main/resources/templates/db_scripts/migration/extract_credits_from_expenditures.sql

```diff
@@ -0,0 +1,59 @@
+-- Extract credits from expenditures in each budget
+-- 1. Calculate all credits values within budget fiscal year
+-- 2. Set to budget credits fields
+-- 3. Subtract from expenditures amount
+
+WITH credit_sums AS (
+    SELECT
+        budget.id AS budget_id,
+        SUM((trx.jsonb->>'amount')::numeric) AS total_credits
+    FROM ${myuniversity}_${mymodule}.budget AS budget
+    JOIN ${myuniversity}_${mymodule}.transaction AS trx
+        ON trx.jsonb->>'fiscalYearId' = budget.jsonb->>'fiscalYearId' AND
+		trx.jsonb->>'fromFundId' = budget.jsonb->>'fundId'
+    WHERE trx.jsonb->>'transactionType' = 'Credit'
```

Comment:
for this one we are good, custom indexes were added to support this query in scope of this change:
https://github.com/folio-org/mod-finance-storage/pull/412/files#diff-4db9b6a16c40e8569cf1f57856808c27dff44495be574ae83559974077c193e4R423